scenarioo-client/test/protractorE2E/webPages/generalSettingsPage.ts

```diff
@@ -22,7 +22,7 @@ util.inherits(GeneralSettingsPage, BaseWebPage);
 
 GeneralSettingsPage.prototype.assertSearchEndpointConfiguredAndReachable = function () {
     expect(this.searchNotConfiguredMessage.isDisplayed()).toBeFalsy();
-    expect(this.configuredSearchEndpoint.getText()).toBe('localhost:9305');
```

Comment:
Yes I know, just wanted to see what happened next in the test, since it is still not working locally.
Just got an update from IT that they distributed a new Pulse Client recently I'll do a retest next week.

Reply:
cool :+1:

Reply:
I don't understand the two changes in this PR. I agree that the port should be changed to the default port 9300. But why did you also change the host from "localhost" to "elasticsearch"?

Reply:
As far as I understood it: Both Tomcat and Elasticsearch run inside the same Docker container. In the Docker configuration we defined, that Elasticsearch can be reached under the alias "elasticsearch".
Maybe @cgrossde can be more specific.